README.md

```diff
@@ -2,10 +2,29 @@
 
 Website for the UBC Computer Science Student Society.
 
+## Installation
+
+- [install Hugo](https://gohugo.io/installation/) - to build the website. The current version of Hugo used is listed in the [`netlify.toml`](netlify.toml) file, under the key `HUGO_VERSION`.
+- [install Node](https://nodejs.org/) - to install npm
+- [install Yarn](https://classic.yarnpkg.com/lang/en/docs/install/) - to install dependencies and run scripts
+
+```bash
+# clone the repository
+git clone https://github.com/ubccsss/ubccsss.org.git
+
+# cd into the repository
+cd ubccsss.org
+
+# install dependencies
+yarn install
+
+# start the development server
+hugo server
+```
+
 ## Contributing
 
-We welcome contributions of any kind including pages, suggestions, bug reports,
-pull requests etc. We would love to hear from you.
+We welcome contributions of any kind including pages, suggestions, bug reports, pull requests etc. We would love to hear from you.
```

Comment:
Could you add a line here along the lines of:

When you submit a PR, we automatically run the [Prettier](https://prettier.io/) code formatter with the [prettier-plugin-go-template](https://github.com/NiklasPor/prettier-plugin-go-template) plugin to help with Go template formatting. Please ensure you checkout the latest version of your PR after any formatting commits have been made when continuing to work on your PR.

just so people aren't confused as to why a commit is automatically made + they also won't run into issues if we request changes?

Reply:
Cool, do we want to create a new commit or amend the commmit instead? @kewbish

Reply:
Creating a new commit will be cleaner IMO because contributers can simply `git pull` instead.